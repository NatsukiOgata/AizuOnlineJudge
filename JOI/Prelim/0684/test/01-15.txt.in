49 51
12 5 30 86 31 34 8 52 30 79 99 46 47 69 63 54 31 76 30 68 14 88 97 6 53 62 97 95 94 42 62 88 53 35 60 86 2 50 69 92 13 4 39 2 30 22 43 28 34
51 67 4 100 9 86 47 82 29 74 14 73 41 73 60 69 78 92 73 74 6 73 79 11 21 41 89 10 49 21 52 59 23 18 57 46 58 84 42 4 55 45 43 77 28 12 26 60 32 14 14
