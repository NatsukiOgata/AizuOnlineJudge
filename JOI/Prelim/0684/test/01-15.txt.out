4
6
12
14
28
42
43
46
47
52
60
69
79
86
92
